fe/fe-core/src/main/java/com/starrocks/sql/analyzer/FunctionAnalyzer.java

```diff
@@ -167,6 +167,12 @@ private static void analyzeBuiltinAggFunction(FunctionCallExpr functionCallExpr)
             }
         }
 
+        if (fnName.getFunction().equals(FunctionSet.ARRAY_SORTBY)) {
+            if (functionCallExpr.getChildren().size() != 2) {
+                throw new SemanticException("array_sortby only support 2 parameters");
```

Comment:
In your example, lambda version could support three parameters ?

Reply:
lambda version will be written to the 2-parameter version, e.g., the lambda function will output an array as the second parameter for the 2-parameter version.